### ISSUE TITLE
Pattern problem ?

### ISSUE THREAD

**Issue by JTriggerFish** [gibber-cc/tidal.pegjs#3]
Is this expected that this sequence works fine
'<0.5 .5*2>'
but this one doesn't
'<.5*2>'

 ( at least in gibber v2 ) ?
It looks like the parser doesn't expect the first atom to have a multiplier ?

**Comment by charlieroberts** [comment]
No, that sounds like a bug, and I doubt I would have written any tests for the step operator with only one member. Thanks for the report!

**Comment by charlieroberts** [comment]
@mariana0pachon I think this is in the PEG, any interest in tackling it?
